satpy/dependency_tree.py

```diff
@@ -60,13 +65,17 @@ def leaves(self, nodes=None, unique=True):
                     res.append(sub_child)
         return res
 
-    def trunk(self, nodes=None, unique=True):
+    def trunk(self, nodes: Optional[IterOfNodes] = None,
+              unique: bool = True,
+              limit_to: Optional[List[DataID]] = None):
         """Get the trunk nodes of the tree starting at this root.
 
         Args:
-            nodes (iterable): limit trunk nodes to the names specified or the
+            nodes: limit trunk nodes to the names specified or the
                               children of them that are also trunk nodes.
             unique: only include individual trunk nodes once
+            limit_to: limit children to the nodes with names in this list
```

Comment:
There is an inconsistency between the `nodes` and `limit_to` argument names. It should be `nodes`/`children` or `limit_nodes_to`/`limit_children_to` iiuc.

Reply:
I went with limit_nodes_to/limit_children_to. Let me know what you think.